roles/cdi/tasks/provision.yml

```diff
@@ -0,0 +1,18 @@
+---
+# General CDI Deployment
```

Comment:
I don't know this thing at all, but this comment makes me curious if there is any other, non-general CDI Deployment?

Reply:
fixed